sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/data/ViewportDataSourceOptions.kt

```diff
@@ -0,0 +1,108 @@
+package com.mapbox.maps.plugin.viewport.experimental.data
+
+import android.location.Location
+
+/**
+ * Describe the viewport data source config options.
+ */
+data class ViewportDataSourceOptions(
```

Comment:
This cannot be a data class, each future addition would break compatibility. Same below.

Reply:
The usage of data class has been removed from this PR, Builder pattern is used instead.